sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Form Recognizer client library for Python
```

Comment:
are we keeping this as is?

Reply:
I'm following what Text Analytics did for their rebranding, keep the client library name the same and only update the references to the service. Makes sense since the package isnt going to be renamed.